tests/components/ct-solid/playwright/index.tsx

```diff
@@ -1,12 +1,14 @@
 import '../src/assets/index.css';
 import { beforeMount, afterMount } from '@playwright/experimental-ct-solid/hooks';
+import { Router } from "@solidjs/router";
 
 export type HooksConfig = {
   route: string;
 }
 
-beforeMount<HooksConfig>(async ({ hooksConfig }) => {
+beforeMount<HooksConfig>(async ({ hooksConfig, Component }) => {
   console.log(`Before mount: ${JSON.stringify(hooksConfig)}`);
+  return <Router><Component /></Router>;
```

Comment:
@pavelfeldman Thinking out loud: Maybe it's useful that the plugins/wrappers for the JSX frameworks can be registered globally in the beforeEach hook, as this is also possible with Vue. This ensures that the tests themselves are less framework specific.